docs/content/api-reference.md

```diff
@@ -90,32 +90,34 @@ Appear is a component that makes a component animate on the slide on key press.
+CodePane is a component for showing a syntax-highlighted block of source code. It will scroll for overflow amounts of code, trim whitespace and normalize indents. It will also wrap long lines of code and preserve the indent. CodePane uses the [React Syntax Highlighter](https://github.com/react-syntax-highlighter/react-syntax-highlighter) Component.
 
-Additionally, `highlightStart` and `highlightEnd` props can be used to highlight certain ranges of code. Combine this with the [Stepper](#stepper) component to iterate over lines of code as you present.
+The `theme` prop accepts a configurable object or a string of the available [Prism Themes](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/src/styles/prism/index.js); `'atomDark'`, `'base16AteliersulphurpoolLight'`, `'cb'`, `'coy'`, `'darcula'`, `'dark'`, `'duotoneDark'`, `'duotoneEarth'`, `'duotoneForest'`, `'duotoneLight'`, `'duotoneSea'`, `'duotoneSpace'`, `'funky'`, `'ghcolors'`, `'hopscotch'`, `'okaidia'`, `'pojoaque'`, `'prism'`, `'solarizedlight'`, `'tomorrow'`, `'twilight'`, `'vs'` and `'xonokai'`.
 
-| Props            | Type                                                                                       | Example               |
-| ---------------- | ------------------------------------------------------------------------------------------ | --------------------- |
-| `autoFillHeight` | PropTypes.boolean                                                                          | `false`               |
-| `children`       | PropTypes.string                                                                           | `let name = "Carlos"` |
-| `fontSize`       | PropTypes.number                                                                           | `16`                  |
-| `highlightEnd`   | PropTypes.number                                                                           | `2`                   |
-| `highlightStart` | PropTypes.number                                                                           | `1`                   |
-| `indentSize`     | PropTypes.number                                                                           | `2`                   |
-| `language`       | PropTypes.string                                                                           | `javascript`          |
-| `theme`          | [Prism Theme](https://github.com/FormidableLabs/prism-react-renderer/tree/main/src/themes) | —                     |
+Additionally, the `highlightRanges` prop accepts an array that can be used to highlight certain ranges of code. This array can contain a range of two numbers `[1, 3]` which will highlight lines 1 through 3. It can also contain a list of sub-arrays which will be considered as a list of ranges `[[1, 3], [6, 8], [10, 15]]`, and if in between the list of ranges, you would like to only highlight a single line, then a number can be included within this list `[[1, 3], 5, [6, 8], [10, 15], 20]`.
```

Comment:
I would add a note that each range is a step in the slide's animation count. They are displayed with each advance.

Reply:
Suggest to minorly change:

```md
, and if in between the list of ranges, you would like to only highlight a single line, then a number can be included within this list `[[1, 3], 5, [6, 8], [10, 15], 20]`.
```

with

```md
. Array values can even be mixed to include sub-arrays (for multiple lines) and numbers (for single lines), e.g., `[[1, 3], 5, [6, 8], [10, 15], 20]`.
```